Classes/Command/LanguageCommandController.php

```diff
@@ -0,0 +1,126 @@
+<?php
+namespace Helhum\Typo3Console\Command;
+
+/*
+ * This file is part of the TYPO3 console project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read
+ * LICENSE file that was distributed with this source code.
+ *
+ */
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Mvc\Controller\CommandController;
+use TYPO3\CMS\Lang\Service\TranslationService;
+
+/**
+ * Language API Command Controller
+ *
+ */
+class LanguageCommandController extends CommandController
+{
+
+    /**
+     * @var \TYPO3\CMS\Lang\Domain\Repository\ExtensionRepository
+     * @inject
+     */
+    protected $extensionRepository;
+
+    /**
+     * @var \TYPO3\CMS\Lang\Service\TranslationService
+     * @inject
+     */
+    protected $translationService;
+
+    /**
+     * Update translations for $languages
+     *
+     * @param string $languages comma separated language value
```

Comment:
this can be annotated as array. The console will take care of converting the comma separated string into an array. (see multiple other commands)